tmate/device.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import os, sys
+from base64 import b64encode
+
+from socketio import Client
+
+username = os.environ['DEVICENAME'].encode('latin1')
```

Comment:
Is there a reason why you are choosing to decode using `latin1` rather then the default `utf-8`?

Reply:
Not really, that's just the result of working with a lot of binary data that needs to be converted to and from strings (for example the call to b64encode). It should probably be `utf-8`, at least for the two `os.environ` lines. For what it's worth, HTTP headers are `latin-1`.

Are you planning on using Unicode characters for device credentials?